src/TraceEvent/Symbols/SymbolReader.cs

```diff
@@ -428,6 +435,10 @@ public string SymbolCacheDirectory
             }
         }
         /// <summary>
+        /// Authorization header to be ued when making requests to source server (only for SourceLink)
+        /// </summary>
+        public string AuthorizationHeaderForSourceLink { get; set; }
```

Comment:
I think this is OK for now.  If we run into more of these, I might like to remove this and provide a callback for modifying the ```HttpClient``` instead.